package.json

```diff
@@ -17,7 +17,8 @@
     "cover": "nyc npm run unit-test && nyc check-coverage --lines 90 --branches 90 --statements 90 --functions 90",
     "lint": "tslint -p tsconfig.json && tslint -p test/tsconfig.json && tslint -p e2e/tsconfig.json",
     "e2e": "mocha --config ./e2e/.mocharc.json \"e2e/**/*.spec.*\"",
-    "driver": "npx ts-node ./driver/ChannelApeClientDriver.ts"
+    "driver": "npx ts-node ./driver/ChannelApeClientDriver.ts",
+    "local": "node_modules/ts-node/dist/bin.js ./src/test.ts"
```

Comment:
any reason, you just didn't utilize the driver here? That's essentially what's it for

Reply:
@ryankazokas this is what I am using for my local script when talking with supabase. I needed a way to see what was coming back from supabase calls. I will be reverting this file when done.

Reply:
Yea, that is what the driver test is. Too make actual external calls without mocking. It's really the only way to test this standalone. I konw driver is a bit of an antiquated term, so i'm fine with you leaving in and just calling local or manual or something along the lines. Just remove the driver test if you decide to that.